v1.0-known-limitations.md

```diff
@@ -0,0 +1,38 @@
+---
+title: v1.0 Known Limitations
+summary:
+toc: false
+---
+
+This page describes limitations we've identified in the v1.0 release and plan to investigate and possibly resolve in a future release.
+
+<div id="toc"></div>
+
+## `INSERT ON CONFLICT` vs. `UPSERT`
+
+When inserting/updating all columns of a table, and the table has no secondary indexes, it's recommended to use an `UPSERT` statement instead of the equivalent `INSERT ON CONFLICT` statement. Wherease the `INSERT ON CONFLICT` always has to read to determine the necessary writes, the `UPSERT` statement writes without first reading and so will be faster.
+
+This limitations may be particularly relevant if you are using a simple SQL table of two columns to [simulate direct KV access](frequently-asked-questions.html#can-i-use-cockroachdb-as-a-key-value-store). In that case, be sure to use the `UPSERT` statement.
+
+## `unique_rowid()` values in Javascript
+
+The `unique_rowid()` [function](functions-and-operators.html) returns values that cannot be represented exactly by numbers in Javascript. For example, a `235191684988928001` value returned by `unique_rowid()` would get represented as `235191684988928000` in Javascript. Notice that the last digit is different. In such cases, if the Javascript application temporarily stores the value `235191684988928001` and queries using it, it won't match the value that actually got stored.
+
+## Repeated commands in the SQL shell history
+
+In the [built-in SQL shell](use-the-built-in-sql-client.html), each previously executed command is stored in the SQL shell history. In cases where identical commands are executed consecutively, this can make it incoventient to return to a command prior to the identical batch of commands.
+
+## Using `\|` to perform a large input in the SQL shell
+
+In the [built-in SQL shell](use-the-built-in-sql-client.html), using the [`\|`](use-the-built-in-sql-client.html#sql-shell-commands) operator to perform a large number of queries from a file can cause the server to close the connection. This is because `\|` sends the entire file as a single query to the server, which can exceed the upper bound on the size of a packet the server can accept from any client (16MB).
+
+As a workaround, [execute the file from the command line](use-the-built-in-sql-client.html#execute-sql-statements-from-a-file) instead of from within the interactive shell.
```

Comment:
I would be more concrete here and say `[ execute ... ]( ... ) with "cat data.sql | cockroach sql", instead ...`